server/sonar-web/src/main/js/apps/projectActivity/components/ProjectActivityAnalysesList.js

```diff
@@ -91,7 +93,7 @@ export default class ProjectActivityAnalysesList extends React.PureComponent {
     this.updateStickyBadges(forceBadgeAlignement);
   };
 
-  scrollToDate = (targetDate: ?Date) => {
+  scrollToDate = (targetDate /*: ?Date */) => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove parentheses around the parameter of this arrow function. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=javascript%3AS3524)